backend/api/Database/Models/MissionDefinition.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+#pragma warning disable CS8618
+namespace Api.Database.Models
+{
+    public class MissionDefinition
+    {
+        [Key]
+        [Required]
+        [MaxLength(200)]
+        public string Id { get; set; }
```

Comment:
Why is this one set by us and not database generated like the others? 

I think it should be generated by the database to ensure uniqueness. Searching for `MissionDefinitions` may be done through an API call which can have options for name, area etc.

Reply:
The only reason I did it this way was because I used the ID when setting the blob URL for the source JSON, and since this URL was inserted into the missionDefinition, it made sense to generate the ID first. I have now changed it so that the source URL is random instead.